docs/sql-ref-syntax-qry-explain.md

```diff
@@ -19,4 +19,126 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `EXPLAIN` statement provides the execution plan for the statement. 
+By default, `EXPLAIN` provides information about the physical plan.
+`EXPLAIN` does not support `DESCRIBE TABLE` statement.
+
+
+### Syntax
+{% highlight sql %}
+EXPLAIN [EXTENDED | CODEGEN | COST | FORMATTED] statement
```

Comment:
Since its better to follow the other docs, `statement` -> `input_statement`? https://github.com/apache/spark/blob/master/docs/sql-ref-syntax-aux-describe-query.md#syntax
Can you check again that kind of consistency for the other statements, too?